### ISSUE TITLE
Improved Handling of Elipsis

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4085]
**Reported by mhorspool on 2014-04-21 01:37**
Announcement of the ellipsis is critical when navigating the Windows environment because it alerts the user that a dialog is about to appear.  NVDA currently does not announce the ellipsis when punctuation is set to 'Some'.  Furthermore, it does pass the ellipsis to the synthesizer, rendering a pause.  This creates an inconsistent user experience because sometimes the control type is immediately spoken after its label, and sometimes a pause is left between the label and the control type.

I will submit that users who have their punctuation set to 'None' are advanced users, and that 'none' means just that, 'none'.  A pause might actually be useful to them because it alerts them that a dialog is about to appear whilst cutting down on verbiage.  However, I would expect "dot dot dot" with no pause at 'Some'.  The punctuation level can be changed in the punctuation dialog box but the pause still exists.

I therefore propose that the preservation flag on the ellipsis be set to 'norep' by default.  I further propose that its default punctuation level be set to some or above.

I have done this at user level and I now hear "Browse dot dot dot button".  Lovely.  Rather than "Browse, button" or "Browse dot dot dot, button", which was most irritating.

**Comment by bhavyashah** [comment]
The elipsis appears in so many dialogs and buttons that I suppose one could consider as essential for default reading. Since my punctuation and symbol level settings are probably non-default, could someone please mention the default level at which ... is spoken (the ticket body seems to suggest None)?

**Comment by ehollig** [comment]
@bhavyashah, my … and ... are set to none. Should this be a topic that is further discussed? Personally, I do not think it should be, as @jcsteh pointed out in https://github.com/nvaccess/nvda/issues/4085#issuecomment-155319634
> Punctuation levels are a fairly controversial topic. However, as I see it, "some" is meant to be best for reading by default. This means it should only read symbols which can't be conveyed by inflection or pause, such as mathematical symbols and the like.

**Comment by bhavyashah** [comment]
@ehollig Thanks for the clarification. Also, upon second thought, I agree with @jcsteh's https://github.com/nvaccess/nvda/issues/4085#issuecomment-155319634. Unless we still have active NVDA users on GitHub who would like to advocate for the original proposal, I would suggest closing this ticket for the time being.

**Comment by ehollig** [comment]
Closing for now. We can reopen if this topic is brought up again.
